mono/metadata/oop.c

```diff
@@ -425,6 +470,11 @@ mono_unity_oop_get_stack_frame_details(
             frameDetails->assemblyNameLen,
             read_pointer(OFFSET_MEMBER(MonoImage, image, assembly_name)));
 
+	frameDetails->assemblyNameLen = read_nt_string(
```

Comment:
I think this is a copy/paste error, and you want to assign to `assemblyFileNameLen`?